docs/en/sql-reference/operators/index.md

```diff
@@ -188,6 +188,24 @@ SELECT now() AS current_date_time, current_date_time + INTERVAL '4' day + INTERV
 └─────────────────────┴────────────────────────────────────────────────────────────┘
 ```
 
+You can work with dates without using `INTERVAL`, just by adding or subtracting seconds, minutes, and hours. For example, an interval of one day can be set by adding `60*60*24`.
+
+!!! note "Note"
+    The `INTERVAL` syntax or `addDays` function are always preferred. Simple addition or subtraction (syntax like `now() + ...`) doesn't consider time settings. For example, daylight saving time.
+
+
+Examples:
+
+``` sql
+SELECT now() AS current_date_time, current_date_time + 60*60*96;
```

Comment:
What if we add better example with time shift?
```sql
:) SELECT toDateTime('2014-10-26 00:00:00', 'Europe/Moscow')  AS time, time + 60*60*24 as time_plus_24_hours, (time + interval 1 day) as time_plus_1_day

SELECT
    toDateTime('2014-10-26 00:00:00', 'Europe/Moscow') AS time,
    time + ((60 * 60) * 24) AS time_plus_24_hours,
    time + toIntervalDay(1) AS time_plus_1_day

Query id: aaf32215-aa05-4a37-a370-b85b73aaf386

┌────────────────time─┬──time_plus_24_hours─┬─────time_plus_1_day─┐
│ 2014-10-26 00:00:00 │ 2014-10-26 23:00:00 │ 2014-10-27 00:00:00 │
└─────────────────────┴─────────────────────┴─────────────────────┘
```

(`2014-10-26 02:00:00` msk was time shift -1 hour)

Reply:
Great example, thanks.